administration/src/project-configs/koblenz/dataPrivacyBase.tsx

```diff
@@ -1,6 +1,7 @@
 import { ReactElement } from 'react'
 
-export const dataPrivacyBaseHeadline = 'Datenschutzerklärung für die Nutzung und Beantragung des digitalen Koblenz-Pass'
+export const dataPrivacyBaseHeadline =
+  'Datenschutzerklärung für die Nutzung und Beantragung des digitalen KoblenzPasses'
```

Comment:
I think we are inconsistent with `KoblenzPass` vs `KoblenzPasses`, i.e. whether to adjust it based on cases.

Reply:
i think since this is a "Eigenname" `KoblenzPass` should be fine. thx for the hint